### ISSUE TITLE
Out-of-date TaxBrain code in test_celery_tasks.py

### ISSUE THREAD

**Issue by martinholmer** [OpenSourcePolicyCenter/PolicyBrain#677]
Looking through the PolicyBrain source code, I found the following in the
`deploy/taxbrain_server/tests/test_celery_tasks.py` file:
```
@contextlib.contextmanager
def patch_taxcalc_dropq():
    old1 = ct.taxcalc.dropq.calculate_baseline_and_reform
    old2 = ct.taxcalc.dropq.groupby_means_and_comparisons
    old3 = ct.taxcalc.dropq.pd.DataFrame
    old4 = ct.taxcalc.dropq.create_json_table
    try:
        def do_nothing(n, *args, **kwargs):
            if n == 'as_dict':
                return {'abc': [1]}
            return tuple(mock.MagicMock() for _ in range(n))
        ct.taxcalc.dropq.calculate_baseline_and_reform = partial(do_nothing, 3)
        ct.taxcalc.dropq.groupby_means_and_comparisons = partial(do_nothing, 19)
        ct.taxcalc.dropq.pd.DataFrame = partial(do_nothing, 1)
        ct.taxcalc.dropq.create_json_table = partial(do_nothing, 'as_dict')
        yield True
    finally:
        ct.taxcalc.dropq.calculate_baseline_and_reform = old1
        ct.taxcalc.dropq.groupby_means_and_comparisons = old2
        ct.taxcalc.dropq.pd.DataFrame = old3
        ct.taxcalc.dropq.create_json_table = old4
```
Not sure if these tests ever get run, but the dropq hasn't had a `calculate_baseline_and_reform` or a `groupby_means_and_comparisons` member for quite some time.  Perhaps some clean-up is in order.

**Comment by MattHJensen** [comment]
@martinholmer, thanks very much for finding this area for improvement. Could you please update the issue name to something more specific and informative? Perhaps something like, "out of date dropq calls in test_celery_tasks.py"

**Comment by hdoupe** [comment]
Thanks for pointing this out @martinholmer. What we really need is a test suite for the flask/celery backend. Currently, the tests have fallen completely out of date. This has been on the radar for a month or two. I'll open up a separate issue to get the ball rolling on this.

**Comment by hdoupe** [comment]
Resolved via #910
